web/views.py

```
@@ -3115,470 +3120,59 @@ def get_object(self):
+def certificate_detail(request, uuid):
+    """
+    View function to display details of a certificate.
+    Allows the owner to view, get embed code, and manage their certificate.
+    """
+    certificate = get_object_or_404(Certificate, uuid=uuid)
 
-    except Donation.DoesNotExist:
-        # This might be a subscription for something else
-        pass
+    # Check if user has permission to view the certificate
+    if not certificate.user == request.user and not request.GET.get("embed", False):
```

Ruff (0.8.2)

Use `certificate.user != request.user` instead of `not certificate.user == request.user`

Replace with `!=` operator

(SIM201)